build/DevDivVsix/EditorsPackage/Microsoft.VisualStudio.Editors.swixproj

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think we should move all of these to projects to the src directory - that's where the source for the other vsixes live. The build folder doesn't seem appropriate for these.